osutil/kcmdline.go

```diff
@@ -203,3 +203,12 @@ func KernelCommandLineKeyValues(keys ...string) (map[string]string, error) {
 	}
 	return m, nil
 }
+
+// KernelCommandLine returns the command line reported by the running kernel.
```

Comment:
not needed for this PR, but we could simplify `KernelCommandLineKeyValues` to use this helper now instead of manually doing exactly the same thing

Reply:
+1